ports/verticaldropheroeshd/README.md

```diff
@@ -0,0 +1,16 @@
+## Notes
+
+I'd like to thank the following:
+- [Nerdok Productions](https://nerdook-productions.com/), for making a fun game;
+- Fraxinus88, for making a killer tutorial on porting Game Maker game and for the code for extrating the .exe;
```

Comment:
Small spelling mistake here,"extrating"

Reply:
Actually that shouldn't be there at all, lol. That's a left over from another port. 😄